metadata-ingestion/examples/recipes/dbt_cloud_sample_to_datahub.dhub.yaml

```diff
@@ -0,0 +1,16 @@
+source:
+    type: dbt-cloud
+    config:
+        project_id: '277462'
+        target_platform: postgres
+        stateful_ingestion:
+            enabled: false
+        account_id: '107298'
+        job_id: '399798'
```

Comment:
can you merge this stuff into the dbt sample recipe instead https://github.com/datahub-project/datahub/blob/05a0f3e2a67028249735e522c385df6306e01e7d/metadata-ingestion/docs/sources/dbt/dbt-cloud_recipe.yml#L12